manifests/system/vmware.pp

```diff
@@ -29,33 +29,25 @@
                       '--output_dir', $metrics_output_dir,
                       '> /dev/null'].join(' ')
 
+  $tidy_command = "${puppet_metrics_collector::system::scripts_dir}/metrics_tidy -d ${metrics_output_dir} -r ${retention_days}"
+
   if ($metrics_ensure == 'present') and (!$facts.dig('puppet_metrics_collector', 'have_vmware_tools')) {
     notify { 'vmware_tools_warning':
       message  => 'VMware metrics collection requires vmware-toolbox-cmd to be on the PATH',
       loglevel => warning,
     }
-    # Set cron job to absent to avoid spamming mailboxes with errors.
-    $_cron_ensure = 'absent'
-  } else {
-    $_cron_ensure = $metrics_ensure
   }
 
-  cron { 'vmware_metrics_collection':
-    ensure  => $_cron_ensure,
-    command => $metrics_command,
-    user    => 'root',
-    minute  => "*/${collection_frequency}",
+  puppet_metrics_collector::collect {'pe_vmware':
```

Comment:
`pe_vmware` implies that this will be a PE-only collection. OSP customers can use this, so maybe `puppet_vmware`?

Reply:
:+1: